test/system/home_page_test.rb

```diff
@@ -8,9 +8,17 @@ class HomePageTest < ApplicationSystemTestCase
   end
 
   test "renders custom hero and about text from PageText or default text" do
-    PageText.create(hero: "Super Pets for a good paws", about: "All about us")
+    create(:page_text, hero: "Super Pets for a good paws", about: "All about us")
+
     visit home_index_path
     assert_text "Super Pets for a good paws"
     assert_text "All about us"
+
```

Comment:
I wasn't sure if this was the best place to test this. Happy to put it elsewhere if there's a better place for it.

Reply:
Yeah that makes sense, but I always worry about adding extra cases for a system test just given how slow they are comparatively. I think you're right that there's not much benefit here